pkg/driverbuilder/builder/amazonlinux.go

```diff
@@ -314,14 +312,3 @@ func bunzip(data io.Reader) (res []byte, err error) {
 
 	return
 }
-
-func amazonLLVMVersionFromKernelRelease(kr kernelrelease.KernelRelease) string {
```

Comment:
We do not allow to switch on llvm/clang version anymore, as it really makes no sense.